src/shared/helpers/formatters/date.ts

```diff
@@ -51,6 +51,18 @@ export function formatTimestamp(
 		.format(`DD-MM-YYYY HH:mm${includeSeconds ? ':ss' : ''}`);
 }
 
+export function formatVerboseTimestamp(
```

Comment:
Would changing `formatTimestamp` to be:
```
formatTimestamp(
  timestamp: DateLikeNullable,
  config: {
    includeSeconds: boolean,
    dashed: boolean
  } = {
    includeSeconds: true,
    dashed: true
  }
): string {...}
```
not be a better idea (+ separating above type & default config)?

Reply:
Ik pas het nu aan naar formatDefaultTimestamp(timestap, format)
waarbij je de formatstring moet meegeven; in dit geval `DD MMMM YYYY HH:mm`, want het gaat om meer dan alleen seconden en dashes, hier moet plots de maand voluit geschreven zijn 🤷